sub-projects/device/tests/test_aws_s3.py

```diff
@@ -0,0 +1,16 @@
+# NOTE: this file must be in the same directory as aws.py and cameron.jpg to around
+from aws import S3
+
+s3_session = S3();
+s3_session.upload('cameron.jpg') # takes the image and stores it into the checkfaces bucket
+s3_session.view() # returns all object from checkfaces bucket
+s3_session.delete('cameron.jpg') # delete the image from checkfaces bucket
+s3_session.view() # returns all objects from checkfaces bucket
+
+#If test works correctly it will return a error at the end saying:
```

Comment:
Does it error out because of line 8 you call `view()` again after you deleted the image from the bucket?